rewrite-maven/src/test/java/org/openrewrite/maven/MavenParserTest.java

```diff
@@ -3738,19 +3733,15 @@ void propertyFromMavenConfig() {
     @Test
     void propertyFromMavenConfigFromParentPomCanBeUsedInChild() {
         rewriteRun(
+          spec -> spec.parser(MavenParser.builder().property("revision", "1.0.0")),
```

Comment:
Though I agree with this kind of change, I still want a test to prove the maven-parser can handle `maven.config` files. Can we still make such a test work?

Reply:
That should be tested in the Maven plugin, if at all, as it's just a plain text file. No need to test that here.